js/faq.js

```diff
@@ -1,8 +1,34 @@
 const faqLinks = document.querySelectorAll('.faq_link');
 const currentLocation = window.location.hash;
 
+window.addEventListener('keydown', (e) => {
+  const ancTag = document.querySelector(
+    `a[href="${e?.target?.attributes?.href?.value}"]`,
+  );
+  if (e.key === ' ' && ancTag) {
+    e.preventDefault();
+    ancTag?.click();
+  }
+});
+
+const removeForOthers = (target) => {
+  faqLinks.forEach((faqLink) => {
+    if (faqLink.getAttribute('href') != target.getAttribute('href')) {
+      const faqText = faqLink.nextElementSibling;
+      const ancTag = faqText.querySelectorAll(`a`);
+      ancTag.forEach((element) => {
+        element.setAttribute('tabindex', '-1');
```

Comment:
since `tabindex` is used in multiple places. can we import it from a constant?

Reply:
we cannot put tabindex into a constant as we change tabindex  frequently for different <a>

Reply:
changed